modules/cosmwasm-cli/cmd/pauser.go

```diff
@@ -75,11 +75,44 @@ func pauserExecute() *cobra.Command {
 
 	command.AddCommand(&cobra.Command{
 		Use:   "pause",
+		Short: "To pause contract's methods in the ecosystem.",
+		Args:  cobra.ExactArgs(2),
+		Run: func(cmd *cobra.Command, args []string) {
+			executeMsg := pauser.ExecuteMsg{
+				Pause: &pauser.Pause{
+					Contract: args[0],
+					Method:   args[1],
+				},
+			}
+
+			clientCtx := sdk.WithKeyring(sdk.NewClientCtx())
+
+			opts := sdk.DefaultBroadcastOptions().
+				WithContractAddr(cmd.Flag("contract").Value.String()).
+				WithExecuteMsg(executeMsg)
+
+			response, err := cosmwasmapi.Execute(
+				clientCtx,
+				context.Background(),
+				clientCtx.GetFromAddress().String(),
+				opts,
+			)
+
+			if err != nil {
+				panic(err)
+			}
+
+			fmt.Printf("Transaction hash: %s\n", response.Hash.String())
+		},
+	})
+
+	command.AddCommand(&cobra.Command{
+		Use:   "global-pause",
```

Comment:
```suggestion
		Use:   "pause-global",
```

Shouldn't this by pause-global?